lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -1503,5 +1527,61 @@ class StandardAccountTest {
             assertEquals(0, messages[0].order)
             assertEquals(1, messages[1].order)
         }
+
+        @Test
+        @Disabled("Used as example in docs")
```

Comment:
A little bit unsure about this, this code is not runnable anyway due to `TypedData.fromJsonString("...")` which will fail. 

Is there any advantage to keeping this inside test file? Or was the idea just to ensure correct syntax?

Reply:
Ok, actually I removed this test and placed is as code block in the guide. In future, we may put all code blocks as example tests, to maintain clarity when class/method changes (I've created https://github.com/software-mansion/starknet-jvm/issues/518 for this).